charts/fybrik/values.yaml

```diff
@@ -442,6 +442,13 @@ opaServer:
   # Bootstrap policies to load upon startup
   bootstrapPolicies: {}
 
+  # Controls if to allow by default or not.
+  # Set DefaultDecision: {} for deny by default.
+  defaultDecision:
```

Comment:
I see a couple of issues here:

1. we increase the values file, theoretically we should provide an explanation to the entries
2. the default behavior should be a boolean: `true` - Allow, `false` - Deny. 
The rego transformation can be done behind of scene in `_helpers.tpl`

@shlomitk1 , can we remove the `bootstrapPolicies` ? slightly simplify the values file.

Reply:
Yes, we can.